src/app/patient-summary/components/generics/DetailedTable.tsx

```diff
@@ -1,52 +1,65 @@
 import {
-  TableContainer,
-  Paper,
   Table,
   TableHead,
   TableRow,
   TableCell,
   TableBody,
   styled,
+  Typography,
 } from '@mui/material';
 
-import { ArrayCaption } from '@/app/components/typography/ArrayCaption';
 import { TSupportedResource } from '@/types/fhir.types';
 
-import { tableConfig } from './resource.types';
-
 export const StyledTableRow = styled(TableRow)(() => ({}));
 
 export const StyledTableCell = styled(TableCell, {
   shouldForwardProp: (prop) => prop !== 'cellNumber',
 })(({ cellNumber }: { cellNumber: number }) => ({
-  width: `${100 / cellNumber}%`,
+  width: `${100 / (cellNumber || 1)}%`,
+  border: '1px solid #eee',
 }));
 
 export const DetailedTable = <T extends TSupportedResource>({
```

Comment:
Can you double check why do we need to declare T if it's not then used?

Reply:
Yes sure i will double check :rocket: